ows.go

```diff
@@ -182,7 +183,7 @@ func serveWMS(ctx context.Context, params utils.WMSParams, conf *utils.Config, r
 
 		xRes := (params.BBox[2] - params.BBox[0]) / float64(*params.Width)
 		if conf.Layers[idx].ZoomLimit != 0.0 && xRes > conf.Layers[idx].ZoomLimit {
-			out, err := utils.GetEmptyTile("zoom.png", *params.Height, *params.Width)
+			out, err := utils.GetEmptyTile(utils.EtcDir+"/zoom.png", *params.Height, *params.Width)
```

Comment:
I would not put the `.png` files in `utils.EtcDir` (that's for configuration), but in `utils.DataDir`. The `share` directory is for architecture-independent data (and can be shared across architectures in a big, site-wide NFS mounted `/usr/local` file system, eg. `/usr/local/share/emacs`).